tests/test_performance.py

```
@@ -0,0 +1,569 @@
+"""
+Performance and Load Tests for Tidal MCP Server
+
+Tests for performance characteristics, memory usage, concurrency,
+and scalability of the Tidal MCP server components.
+"""
+
+import pytest
+import asyncio
+import time
+import threading
+from concurrent.futures import ThreadPoolExecutor, as_completed
```

Ruff (0.12.2)

`concurrent.futures.ThreadPoolExecutor` imported but unused

Remove unused import

(F401)